docs/diginstroom/sip/1.0/usecases/gigapixel-artwork.md

```diff
@@ -0,0 +1,540 @@
+      <premis:relatedObjectIdentifier>
+        <premis:relatedObjectIdentifierType>VIAA PID</premis:relatedObjectIdentifierType>
+        <premis:relatedObjectIdentifierValue>cf9j41p15z</premis:relatedObjectIdentifierValue>
+      </premis:relatedObjectIdentifier>
+    </premis:relationship>
+  </premis:object>
+
+  <premis:object xsi:type="premis:file">
+
+    <premis:objectIdentifier>
+      <premis:objectIdentifierType>UUID</premis:objectIdentifierType>
+      <premis:objectIdentifierValue>uuid-6A07B2FE-3363-4B46-953E-59A30BE67F9B</premis:objectIdentifierValue>
+    </premis:objectIdentifier>
+
+    <premis:objectCharacteristics>
+      <premis:fixity>
+        <premis:messageDigestAlgorithm authority="cryptographicHashFunctions" authorityURI="http://id.loc.gov/vocabulary/preservation/cryptographicHashFunctions" valueURI="http://id.loc.gov/vocabulary/preservation/cryptographicHashFunctions/md5">
+                MD5
+        </premis:messageDigestAlgorithm>
+        <premis:messageDigest>08d339fdc35d17620992cbeeecaca8eb</premis:messageDigest>
+      </premis:fixity>
+      <premis:size>558891008</premis:size>
+      <premis:format>
+        <premis:formatRegistry>
+          <premis:formatRegistryName>PRONOM</premis:formatRegistryName>
+          <premis:formatRegistryKey>fmt/996</premis:formatRegistryKey>
+          <premis:formatRegistryRole authority="http://id.loc.gov/vocabulary/preservation/formatRegistryRole" valueURI="http://id.loc.gov/vocabulary/preservation/formatRegistryRole/spe">specification</premis:formatRegistryRole>
+        </premis:formatRegistry>
+        <premis:formatNote></premis:formatNote>
+      </premis:format>
+    </premis:objectCharacteristics>
+
+    <premis:originalName>cf9j41p15z_stitch_psb</premis:originalName>
+
+    <!-- relationship between file and its representation -->
+    <premis:relationship>
+      <premis:relationshipType authority="relationshipType" authorityURI="http://id.loc.gov/vocabulary/preservation/relationshipType" valueURI="http://id.loc.gov/vocabulary/preservation/relationshipType/str">structural</premis:relationshipType>
+      <premis:relationshipSubType authority="relationshipSubType" authorityURI="http://id.loc.gov/vocabulary/preservation/relationshipSubType" valueURI="http://id.loc.gov/vocabulary/preservation/relationshipSubType/isi">is included in</premis:relationshipSubType>
+      <premis:relatedObjectIdentifier>
+        <premis:relatedObjectIdentifierType>UUID</premis:relatedObjectIdentifierType>
+        <premis:relatedObjectIdentifierValue>uuid-70dd6ae0-37fe-11ed-9b20-7e92631d7d28</premis:relatedObjectIdentifierValue>
+      </premis:relatedObjectIdentifier>
+    </premis:relationship>
+
+  </premis:object>
+
+</premis:premis>
+```
+
+The folders of representation 1, 2, 3, and 5 have a identical structure.
```

Comment:
If representation 5 has an identical structure, why do you add a separate heading and example for it below?